### ISSUE TITLE
Create a 'published datasets wrangling' SOP

### ISSUE THREAD

**Issue by mshadbolt** [ebi-ait/hca-ebi-wrangler-central#225]
We are missing an SOP on how to go about wrangling a published dataset. 

I have created a stub here to be expanded upon:

https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/docs/SOPs/published_datasets_SOP.md

once this is done we should delete https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/master/docs/SOPs/Wrangling%20standards%20and%20guidelines.md as it is superseded by other SOPs

**Comment by Wkt8** [comment]
After a brief chat with Ami, Enrique, Marion, Ray, we decided it might be better to include the 'published datasets wrangling' SOP together with the existing 'Contributor Dataset SOP', as there is a large amount of overlap between the two SOPs, and this would allow for easier updating. (If the process changes then it is not required to change both SOPs). 

I have added and updated the contributor Dataset SOP (https://ebi-ait.github.io/hca-ebi-wrangler-central/SOPs/contributor_dataset_SOP.html) 

I think there is some useful stuff in the Wrangling_Standards_Guidelines that I want to incorporate into the contributor/published dataset SOP, such as informing the members when the dataset has been finally published in the DCP (not just archiving). 

Still to do: 
- [ ] Add the final date of edit after all the edits are done 
- [ ] Delete the published_datasets_SOP stub
- [ ] Delete the Wrangling standards guidelines 
- [ ] Final edits to contributor dataset SOP
- [ ] Rename the contributor dataset SOP to 'Wrangling datasets SOP'?

**Comment by Wkt8** [comment]
I want to check with another wrangler whether we still do this (from the Wrangling_Standards_Guidelines) at the end of a project. The repo was last added to 2 years ago. 

Move all the corresponding documents to the finished_projects in hca-wrangling repo and to Google Drive/Brokering/PROJECTS-FINISHED

Still to complete: 

- [ ]  Delete the Wrangling standards guidelines
- [ ]  Final edits to contributor dataset SOP

**Comment by Wkt8** [comment]
Checked with @mshadbolt that we should add the 'Move all the corresponding documents to the finished_projects in hca-wrangling repo and to Google Drive/Brokering/PROJECTS-FINISHED' to the Wrangling Datasets SOP. I have added it and deleted the wrangling standards guidelines. 

Finished the actions from this task.
